test/SparqlAntlrParserTestHelpers.h

```diff
@@ -14,22 +14,16 @@
 #include "parser/SparqlParserHelpers.h"
 #include "parser/data/OrderKey.h"
 #include "parser/data/VarOrTerm.h"
+#include "util/GTestHelpers.h"
 #include "util/SourceLocation.h"
 #include "util/TypeTraits.h"
 
-#ifdef AD_PROPERTY
-#error "AD_PROPERTY must not already be defined. Consider renaming it."
-#else
-#define AD_PROPERTY(Class, Member, Matcher) \
-  testing::Property(#Member "()", &Class::Member, Matcher)
-#endif
-
-#ifdef AD_FIELD
-#error "AD_FIELD must not already be defined. Consider renaming it."
-#else
-#define AD_FIELD(Class, Member, Matcher) \
-  testing::Field(#Member, &Class::Member, Matcher)
-#endif
+// The following two macros make the usage of `testing::Property` and
+// `testing::Field` simpler and more consistent. Examples:
+//  AD_PROPERTY(std::string, empty, IsTrue);  // Matcher that checks that
+//  `arg.empty()` is true for the passed std::string `arg`.
+// AD_FIELD(std::pair<int, bool>, second, IsTrue); // Matcher that checks, that
+// `arg.second` is true for a`std::pair<int, bool>`
```

Comment:
Is this a github artifact, or is this comment still there? if so, it should be deleted.